lib/bank_account/checking_account.rb

```diff
@@ -0,0 +1,39 @@
+module BankAccount
+  class CheckingAccount < Account
+
+    attr_accessor :id, :balance, :checks
+
+    def initialize (id, initial_balance)
+      @id = id
```

Comment:
Because this is mostly the same as the Parent, just with added `@checks`, you can utilize `super` to do less typing.
```
def initialize(id, initial_balance)
  super
  @checks = 3
end
```